includes/functions.php

```diff
@@ -2000,6 +2000,8 @@ function device_is_up($device, $record_perf = false)
     $device_perf = $ping_response['db'];
     $device_perf['device_id'] = $device['device_id'];
     $device_perf['timestamp'] = ['NOW()'];
+    $maintenance = DeviceCache::get($device['device_id'])->isUnderMaintenance();
+    $mode = Config::get('graphing.availability_ignore_maintenance');
```

Comment:
variable name should be more clear
maybe: "consider_maintenance mode"

Reply:
Changed to availability_consider_maintenance